docs/src/index.md

```diff
@@ -26,3 +26,50 @@ notebooks in order to showcase this package's functionality.
 ```
 # put example notebooks here
 ```
+
```

Comment:
I'm kind of ambivalent about including this section. I agree it'd be nice if there were a way to get other people in lab up to speed quickly with Julia, but rather than linking to ~20 sections of the Julia docs, you may as well tell others just to study the Julia docs, since so many sections are listed here. It's important to understand Julia to understand how this package works, but that's true of every other package as well. I would suggest either removing this section or slimming it down to highlight the Julia features you found most confusing / different from Python / etc.